_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,142 @@
+---
+
+Managing indexes, aliases, and templates in OpenSearch Dashboards can be difficult using only the API because there are restrictions and relation-binding tasks, such as adding aliases to indexes, simulating an index template by its name, and more. We are pleased to announce that these cluster operations have been significantly simplified with the Index Management UI enhancements in OpenSearch Dashboards v2.5.
+
+## Visual editor for index mappings with nested tree editor
+
+Building index mappings in pure JSON because it has several nested layers and properties. To simplify this task, we now provide a visual editor with editable nested properties, as shown in the following image. You can add properties by choosing the operation buttons and then see what the mappings will look like by switching to the JSON editor.
+
+<img src="/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/index-mapping-editor.png" alt="index mapping editor"/>{: .img-fluid }
+
+## Simulate an index by index name
+
+It is challenging to predict the appearance of the index taking into account existing templates. To resolve this issue, we will attempt to match the index name with the templates whenever you change the index name. We will merge what users manually input with what the matching template contains, making it a "what you see is what you get" index.
+
+<img src="/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/simulate-template.png" alt="simulate template"/>{: .img-fluid }
+
+## Edit settings by JSON editor and diff mode
+
+The new Index Management UI offers a visual editor and a JSON editor, which you can use if the visual editor does not support all available fields. Additionally, we provided the editor with a `diff` mode, as shown in the following image, so that you can compare the existing index and see what changes they have made to ensure that no mistakes are made.
+
+<img src="/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/json-diff-editor.png" alt="JSON diff editor"/>{: .img-fluid }
+
+### One click to manage aliases on indexes
+
+It is straightforward to find out what aliases an index container through API, but it is difficult to determine how many indexes an alias points to through the API. The **Aliases** page provides you with these results by grouping indexes by alias. With the Index Management UI, it will be much effortless to attach or detach indexes from an alias, and the alias actions will be automatically generated, eliminating the need for you to build alias actions to add or remove indexes behind an alias and manually enter the indexes, which can lead to mistakes.
```

Comment:
> Use judgment-free language. Words like _simple_, _easy_, and _just_ create a skill judgment that may not apply to everyone in the OpenSearch community. Avoid Latinisms like "via." In the third sentence forward, will these things be the case in the future (we use future tense), or are they true now because of the enhancements? Reference the image in the text that precedes it.

Restore the previous comment and merge the suggestions.